src/maintenance/azext_maintenance/generated/commands.py

```diff
@@ -52,22 +52,44 @@ def load_command_table(self, _):
-                            client_factory=cf_maintenance_configuration, is_experimental=True) as g:
+                            client_factory=cf_maintenance_configuration) as g:
         g.custom_command('list', 'maintenance_configuration_list')
         g.custom_show_command('show', 'maintenance_configuration_show')
         g.custom_command('create', 'maintenance_configuration_create')
         g.custom_command('update', 'maintenance_configuration_update')
         g.custom_command('delete', 'maintenance_configuration_delete', confirmation=True)
 
+    from azext_maintenance.generated._client_factory import cf_maintenance_configuration_for_resource_group
+    maintenance_maintenance_configuration_for_resource_group = CliCommandType(
+        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._maintenance_configurations_for_resourc'
+        'e_group_operations#MaintenanceConfigurationsForResourceGroupOperations.{}',
+        client_factory=cf_maintenance_configuration_for_resource_group)
+    with self.command_group('maintenance configuration-for-resource-group',
+                            maintenance_maintenance_configuration_for_resource_group,
+                            client_factory=cf_maintenance_configuration_for_resource_group) as g:
+        g.custom_command('list', 'maintenance_configuration_for_resource_group_list')
+
+    from azext_maintenance.generated._client_factory import cf_apply_update_for_resource_group
+    maintenance_apply_update_for_resource_group = CliCommandType(
+        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._apply_update_for_resource_group_operat'
+        'ions#ApplyUpdateForResourceGroupOperations.{}',
+        client_factory=cf_apply_update_for_resource_group)
+    with self.command_group('maintenance applyupdate-for-resource-group', maintenance_apply_update_for_resource_group,
+                            client_factory=cf_apply_update_for_resource_group) as g:
+        g.custom_command('list', 'maintenance_applyupdate_for_resource_group_list')
+
     from azext_maintenance.generated._client_factory import cf_update
     maintenance_update = CliCommandType(
-        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._update_operations#UpdateOperations.{}',
+        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._updates_operations#UpdatesOperations.{'
+        '}',
         client_factory=cf_update)
-    with self.command_group('maintenance update', maintenance_update, client_factory=cf_update,
-                            is_experimental=True) as g:
+    with self.command_group('maintenance update', maintenance_update, client_factory=cf_update) as g:
         g.custom_command('list', 'maintenance_update_list')
         g.custom_command('list-parent', 'maintenance_update_list_parent')
+
+    with self.command_group('maintenance', is_experimental=True):
+        pass
```

Comment:
You should not use `is_experimental=True` if it is GA release.

Reply:
I have regenerated code using --extension-mode=stable flag in autorest cmd it removed  above method.

Please review PR